ct_only.go

```diff
@@ -63,8 +62,8 @@ func convertCTEntry(e *ctonly.Entry) *Entry {
 }
 
 // WithCTLayout instructs the underlying storage to use a Static CT API compatible scheme for layout.
-func WithCTLayout() func(*options.StorageOptions) {
-	return func(opts *options.StorageOptions) {
+func WithCTLayout() func(*AppendOptions) {
```

Comment:
A TODO would be ace here to indicate that this is known to be temporary. This option will need to be passed into every lifecycle state. As discussed in person, maybe moving this down to the drivers is a sensible strategy.